### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by kirielss**

### CODE REVIEW

File: MathGame.kirielss/MathGame.kirielss/Game.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+                case 3:
+                    X = A.Next(MAX);
+                    Y = B.Next(100);
+                    Result = X * Y;
+                    Question = ($"{X} x {Y} = ");
+                    Console.Write(Question);
+                    break;
+
+                case 4:
```
Comment (TheCSharpAcademy) [inline MathGame.kirielss/MathGame.kirielss/Game.cs:53]: image

Hey there! As per requirements, you shouldn't allow your division to result in a split number. You have to create a loop that keeps running until the remainder between the two random numbers = 0. Hope that makes sense?

File: MathGame.kirielss/MathGame.kirielss/Game.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                    Y = B.Next(1,9);
+                    Result = X / Y;
+                    Question = ($"{X} / {Y} = ");
+                    Console.Write(Question);
+                    break;
+                    
+            }
+
+        }
+
```
Comment (TheCSharpAcademy) [inline MathGame.kirielss/MathGame.kirielss/Game.cs:64]: Please remove unnecessary empty lines, as per [code conventions](https://www.thecsharpacademy.com/article/58).

File: MathGame.kirielss/MathGame.kirielss/Game.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
+﻿using System;
```
Comment (TheCSharpAcademy) [inline MathGame.kirielss/MathGame.kirielss/Game.cs:1]: Please remove unused 'usings' as per code conventions:

image

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]
Your project is looking great! And you were very close to completing it. Let me know if you have questions about my comments.

**Comment by TheCSharpAcademy** [comment]
@kirielss Could you please submit the link to your project on the dashboard (or project page?), I saw that you submitted the calculator, but couldn't find a record for the math game.

**Comment by TheCSharpAcademy** [comment]
> hey! this link? https://github.com/kirielss/CodeReviews.Console.MathGame/tree/main/MathGame.kirielss

yes please

**Comment by TheCSharpAcademy** [comment]
> hey! this link? https://github.com/kirielss/CodeReviews.Console.MathGame/tree/main/MathGame.kirielss

Actually, I think you had already submitted? I think I had mistakenly marked it as complete (or something else weird happened), but the project is marked as under review now. I'll mark it as complete as soon as you work on the requested changes 😁

**Comment by TheCSharpAcademy** [comment]
@kirielss Hey there! Just checking how you're going with the project. Do you have questions about the changes requested? 😊

**Comment by kirielss** [comment]
Yeah, i didn't get how you want the division. Can you give some input/output examples?

**Comment by TheCSharpAcademy** [comment]
> Yeah, i didn't get how you want the division. Can you give some input/output examples?
> 
> So it is supposed to run a loop until % 2 == 0, but should the user keep providing answers till the loop is done? (sorry, I tried to create it without watching the video tutorial)

No, the program should generate 2 numbers in the background until it finds numbers that will result in an integer. Then the equation will be presented to the user. 😃

**Comment by TheCSharpAcademy** [comment]
@kirielss Let me know if you need help! 😊

**Comment by TheCSharpAcademy** [comment]
@kirielss  I'm closing the PR for admin purposes but feel free to reopen it if you want to work on the project.

**Pull request closed without merging**